openedx/adg/lms/applications/helpers.py

```diff
@@ -55,20 +55,23 @@ def check_validations_for_past_record(attrs, error_message):
     """
     errors = {}
 
+    started_date = datetime(year=attrs['date_started_year'], month=attrs['date_started_month'], day=1)
+    errors = update_errors_if_future_date(started_date, errors, 'date_started_year')
+
     if not attrs.get('date_completed_month', False):
         errors['date_completed_month'] = _(error_message).format(key='Date completed month')
 
     if not attrs.get('date_completed_year', False):
         errors['date_completed_year'] = _(error_message).format(key='Date completed year')
 
-    started_date = datetime(year=attrs['date_started_year'], month=attrs['date_started_month'], day=1)
-
     if attrs.get('date_completed_month') and attrs.get('date_completed_year'):
         completed_date = datetime(year=attrs['date_completed_year'], month=attrs['date_completed_month'], day=1)
 
         if completed_date <= started_date:
             errors['date_completed_year'] = _('Completion date must comes after started date')
```

Comment:
Can we update this error? I know you have not changed this but I feel like we should update this.
Maybe "Completion date must be greater than started date".

PS: If the above message was not a requirement.